src/documents/ChangeApplierAPI.md

```diff
@@ -235,20 +355,23 @@ It is not currently possible to supply more than one wildcard segment per path r
 
 ### Programmatic style for listening to changes###
 
-The programmatic style for registering interest in model changes uses an API exposed by the ChangeApplier on its member `modelChanged` that is very similar to that exposed by a standard [Infusion Event](InfusionEventSystem.md) - the difference is that the `addListener` method accepts an extra 1st argument, `pathSpec` - this holds the same model path reference documented in the previous section on declarative binding:
+The programmatic style for registering interest in model changes uses an API exposed by the ChangeApplier on its member `modelChanged` that is very similar to that exposed by a standard [Infusion Event](InfusionEventSystem.md) - 
+the difference is that the `addListener` method accepts an extra 1st argument, `spec` - an `Object` which holds the same model path reference in `path` or `segs` documented in the previous section on declarative binding:
 
 ```javascript
-applier.modelChanged.addListener(pathSpec, listener, namespace)
+applier.modelChanged.addListener(spec, listener, namespace)
 applier.modelChanged.removeListener(listener)
 ```
 
+`spec` may also include the standard members `namespace` and `priority` seen in the declarative record.
```

Comment:
When manually calling `applier.modelChanged.addListener()`, the `namespace` can be supplied via: 1. as a property of the `spec` argument; 2. the `namespace` argument. If both are provided, which one will take priority?